### PR TITLE
Run functions locally examples

### PR DESCRIPTION
**Pull request by rJankowski93**
**Description**

Changes proposed in this pull request:

- add exmple to changes from branch run-functions-locally
- add temp section replace in examples/go.mod

**Related issue(s)**

https://github.com/kyma-project/cli/pull/717
https://github.com/kyma-project/kyma/issues/9508

### CODE REVIEW

File: function/examples/docker/run/main.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		ContainerName: "test123",
+		Image:         runtimes.ContainerImage(types.Nodejs12),
+		WorkDir:       "/tmp/tmpfunc/",
+		Commands:      runtimes.ContainerCommands(types.Nodejs12),
+		User:          runtimes.ContainerUser(types.Nodejs12),
+	}
+	cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	if err != nil {
+		fmt.Println(err)
+	}
```
Comment (m00g3n) [inline function/examples/docker/run/main.go:26]: I believe we should [exit](https://golang.org/pkg/os/#Exit) after printing error.

File: function/examples/go.mod [hunk 2]
```diff
@@ -2,11 +2,20 @@ module github.com/kyma-incubator/hydroform/function-examples
 
 go 1.14
 
+//replace github.com/kyma-incubator/hydroform/function => github.com/pPrecel/hydroform/function 2bae0404bc9d1ec28031c64776f9f96768b9e878
+replace github.com/kyma-incubator/hydroform/function => github.com/pPrecel/hydroform/function v0.0.0-20210216092427-2bae0404bc9d
```
Comment (m00g3n) [inline function/examples/go.mod:6]: We need to remember to remove this once the [function PR](https://github.com/kyma-incubator/hydroform/pull/186) will be merged and before we merge `examples`.

File: function/examples/docker/run/main.go [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	ctx := context.Background()
+	docker.RunContainer(ctx, cli, runOpts)
```
Comment (m00g3n) [inline function/examples/docker/run/main.go:28]: We should handle error that is returned by `RunContainer` function.

**Review by m00g3n — COMMENTED** [review]
I believe the files for docker run example command were added by accident on project's root level instead `cmd` directory.

File: function/examples/docker/run/main.go (hunk 1 above)
Comment (rJankowski93) [reply function/examples/docker/run/main.go:28]: Done

File: function/examples/docker/run/main.go (hunk 3 above)
Comment (rJankowski93) [reply function/examples/docker/run/main.go]: Done

**Review by m00g3n — APPROVED** [review]

File: function/examples/go.mod (hunk 2 above)
Comment (rJankowski93) [reply function/examples/go.mod]: Done

File: function/examples/README.md [hunk 4]
```diff
@@ -64,3 +64,11 @@ See how the examples use specific libraries to:
 * [Initialize the workspace locally (`init`)](./cmd/workspace/init/main.go).
 * [Apply your workspace on a cluster (`apply`)](./cmd/workspace/apply/main.go).
 * [Fetch cluster resources to synchronize your local workspace (`sync`)](./cmd/workspace/sync/main.go).
+
+## Docker
+
+Docker allow you to perform operation on the given resources: `run`.
```
Comment (kazydek) [inline function/examples/README.md:70]:
```suggestion
Docker allows you to perform the `run` operation on the given resources.
```

File: function/examples/README.md [hunk 5]
```diff
@@ -64,3 +64,11 @@ See how the examples use specific libraries to:
...
+
+See how the examples use specific libraries to:
+
+* [Run resources](./cmd/docker/run/main.go).
```
Comment (kazydek) [inline function/examples/README.md:74]:
```suggestion
```

File: function/examples/README.md [hunk 6]
```diff
@@ -64,3 +64,11 @@ See how the examples use specific libraries to:
 * [Initialize the workspace locally (`init`)](./cmd/workspace/init/main.go).
 * [Apply your workspace on a cluster (`apply`)](./cmd/workspace/apply/main.go).
 * [Fetch cluster resources to synchronize your local workspace (`sync`)](./cmd/workspace/sync/main.go).
+
+## Docker
+
+Docker allow you to perform operation on the given resources: `run`.
+
+See how the examples use specific libraries to:
```
Comment (kazydek) [inline function/examples/README.md:72]:
```suggestion
See how the examples use specific libraries to [run resources](./cmd/docker/run/main.go).
```

**Review by kazydek — APPROVED** [review]

**Pull request merged**